DockerHubBackend/DockerHubBackend/Security/JwtHelper.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace DockerHubBackend.Security
+{
+    public class JwtHelper : IJwtHelper
+    {
+        private readonly string _key;
+        private readonly string _issuer;
+        private readonly int _expiration; // Expiration of jwt in minutes
+
+        public JwtHelper(IConfiguration configuration)
+        {
+            _key = configuration["Jwt:Key"];
+            _issuer = configuration["Jwt:Issuer"];
+            _expiration = Convert.ToInt32(configuration["Jwt:Expiration"]);
+        }
+        public string GenerateToken(string role, string userId, DateTime? lastPasswordChange)
+        {
+            string lastPasswordChangeStr = lastPasswordChange != null ? lastPasswordChange.ToString() : DateTime.MinValue.ToString("o");
```

Comment:
@miloss01 asks what is this field for

Reply:
namesta me da ispadnem glup

Reply:
Let's say access token lasts for 1 day. In mean time a user changes the password. "LastPasswordChange" makes sure that token is not valid anymore.

Reply:
Should we implement password change functionality (I don't see it in specs)?